spec/models/parent_object_spec.rb

```diff
@@ -3,5 +3,63 @@
 require 'rails_helper'
 
 RSpec.describe ParentObject, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  before do
+    prep_metadata_call
+  end
+  context "a newly created ParentObject with just the oid and default authoritative_metadata_source (Ladybird for now)" do
+    let(:po) { described_class.create(oid: "2004628") }
+    before do
+      stub_request(:get, "https://metadata-api-test.library.yale.edu/metadatacloud/api/ladybird/oid/2004628")
+        .to_return(status: 200, body: File.open(File.join(fixture_path, "ladybird", "2004628.json")).read)
+      po
+    end
+
+    it "pulls from the MetadataCloud for Ladybird and not Voyager or ArchiveSpace" do
+      expect(po.authoritative_metadata_source_id).to eq 1 # 1 is Ladybird
```

Comment:
This might be easier for someone else to quickly parse if it read  
```
LADYBIRD = 1
expect(parent_object.authoritative_metadata_source_id).to eq LADYBIRD
```

If we end up referencing 1 / 2 / 3 for LadyBird, ASpace, Voyager in lots of places, it might even make sense to set up global constants.  OR we could ask the metadata cloud to pass a string ID that came from a semantic vocabulary ["LadyBird", "Voyager", "ASpace"]